platform-sdk/swirlds-base/src/main/java/com/swirlds/base/utility/ToStringBuilder.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.base.utility;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.Objects;
+
+/**
+ * A builder for toString methods.
+ */
+public class ToStringBuilder {
+    private final StringBuilder builder;
+    private final Class<?> thisClass;
+    private final Class<?> superClass;
+    private final int initLength;
+
+    public ToStringBuilder(@NonNull final Object object) {
+        builder = new StringBuilder();
+        thisClass = object.getClass();
+        superClass = thisClass.getSuperclass();
+
+        builder.append(thisClass.getSimpleName()).append("[");
```

Comment:
The original `ToStringBuilder` uses `[` and the default Idea generation uses `{`. 

Since many tests use the exact string I’ve tweaked may manually